### ISSUE TITLE
problems with collecting text variables

### ISSUE THREAD

**Issue by pmcmm** [eirannejad/pyRevit#380]
Hi there,

I'm facing a problem with the variables of the type text that belong to a space. In the example in the screenshot, if I try to print the name of a space called 05. Bedroom I get an error. Funnily, I can print the number 00.02.05 BUT if i change it to A00.02.05 I get the same error. 

There is definitely something up with the ironPython here...

![printscreen pyrevit](https://user-images.githubusercontent.com/12100722/46150375-af69c080-c26c-11e8-8b3a-a748cf49f4a2.png)

**Comment by gtalarico** [comment]
See this issue: `.Name` property and other properties sometimes are dropped in ironpython:
https://github.com/IronLanguages/ironpython2/issues/79#issuecomment-313970079

TLDR, as a workaround use:
`Element.Name.__get__(space)`
or
`Element.Name.GetValue(space)`

**Comment by pmcmm** [comment]
thanks a lot gtalarico, it works!

I would very surprised if such a flagrant issue would have not been solved by now..

**Comment by gtalarico** [comment]
For that shared you should be able to use the regular lookup method:
`space.LookupParameter("BARC_SPC_GROUP_CLASS0").AsString()`
